File: src/main/java/common/OkHttpHelper.java
package common;

import okhttp3.OkHttpClient;
import okhttp3.Request;
import okhttp3.Response;

import java.io.IOException;


public class OkHttpHelper {

    private static OkHttpClient client = new OkHttpClient();

    public static String get(String url) throws IOException {
        Request request = new Request.Builder()
                .url(url)
                .build();

        try {
            Response res = client.newCall(request).execute();
            return res.body().string();
        } catch (IOException e) {
            Log.it(e);
            throw e;
        }
    }

    public static String getWithLog(String url) throws IOException {
        System.out.println("OkHttp call URL = " + url);
        return get(url);
    }

}

File: src/main/kotlin/common/OkHttpHelper_.kt
package common

import okhttp3.OkHttpClient
import okhttp3.Request
import java.io.IOException


class OkHttpHelper_ {

    companion object {

        private val client by lazy { OkHttpClient() }

        @Throws(IOException::class)
        fun get(url: String): String {
            val request = Request.Builder()
                    .url(url)
                    .build()

            try {
                val res = client.newCall(request).execute()
                return res.body().string()
            } catch (e: IOException) {
                Log.it(e)
                throw e
            }

        }

    }

}
